storm-core/src/ui/public/flux.html

```diff
@@ -0,0 +1,139 @@
+<html>
```

Comment:
Sorry I misunderstood. @HeartSaVioR not sure what location in the homepage do we want to put the forward link? 

One option could be to skip the link in homepage and provide it via Ambari View.